### ISSUE TITLE
Accessibility - Pop up window has same text

### ISSUE THREAD

**Issue by jithinjosepkl** [wixtoolset/issues#5994]
* Which version of WiX are you building with?

3.10.3.3010

* Which version of Visual Studio are you building with (if any)?

VS 2017

* Which version of .NET are you building with?

4.0

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

Windows 10

* Describe the problem and the steps to reproduce it.

Popup window (when clicked cancel) has the same text as base window. Please see attached image from accessibility team.

* Describe the behavior you expected and how it differed from the actual behavior.

Texts should be different

![3746167_A11y_MSMPI_MSMPISDKSetupPopup_Screenreader](https://user-images.githubusercontent.com/6690627/57099195-b2c89d00-6cd0-11e9-9cf2-663c7391a979.png)

**Comment by barnson** [comment]
This is controlled via loc string. WiX v4 can change it.

**Comment by AlasdairKing** [comment]
This is sensible behaviour as it is. 

**Potentially-affected users with a disability**

Not Deaf or deaf, physical impairments or cognitive impairments: this is a theoretical problem for blind people who would need to discriminate between the two windows (the main window and the pop-up) but can't see them - screenreader users. 

**Impact**
I can understand that this looks like a problem. Users might be using some kind of mechanism to read out window titles, and these look the same! But it isn't: the messagebox is **modal**. It is entirely correct that it has the same title as the main window, because it is communicating a message from the main window: that this is "Microsoft MPI SDK (10.0.12498.5) Setup" and it has a message for you. Giving it a DIFFERENT title might actually be more confusing!

Examples:

Using my screenreader I trigger a list of all the open windows on the desktop and cursor up and down it. I expect to find the installation I am running, but I miss it because I am listening for "Microsoft MPI SDK (10.0.12498.5) Setup" and it is now called "Cancel Setup".

As a keyboard user, I Alt-Tab away from the installation and read a bit of email while it is installing. I alt-tab back, but I don't hear the "Microsoft MPI SDK (10.0.12498.5) Setup" window I'm expecting - where has it gone? I have to go carefully through each window.

This should be left as it is and not changed. @barnson observes that authors can set the title in their MSI if they have a different scenario.

**Comment by barnson** [comment]
@AlasdairKing, your logic seems sound, at least to this a11y amateur. As anyone who prefers otherwise can override the loc string, I'm closing this issue.
